### ISSUE TITLE
GUI screenshot

### ISSUE THREAD

**Issue by alcortesm** [alcortesm/mazegame#2]
Not really an issue, but a screenshot of the GUI version of the tool.
![gui](https://cloud.githubusercontent.com/assets/9169414/10363390/97d37124-6db4-11e5-8c42-9a0f68d3ff02.PNG)

**Comment by sike25** [comment]
Please, how did you implement the board design? With Java Swing?

**Comment by alcortesm** [comment]
yes, see https://github.com/alcortesm/mazegame/blob/master/src/mazegame/client/gui/Gui.java.
